source/configure-mongosh.txt

```diff
@@ -0,0 +1,31 @@
+.. _mongosh-configure:
+
+=====================
+Configure ``mongosh``
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Read the following pages to learn how to configure ``mongosh`` behaviors
```

Comment:
[Suggestion]

You could consider editing this down to "Learn how to configure ``mongosh`` behaviors and user experience:".